src/app/core/parameters/pointArray.ts

```
@@ -0,0 +1,30 @@
+import Parameter from "../parameter";
+import Registry from "../registry";
+
+export default class PointArray extends Parameter {
+    typeString: string = "PointArray";
+    description: string = "Should be an array of points";
+
+    constructor(value: Array<Array<number>>) {
+        super("PointArray",value);
+    }
+
+    isValid(value: any) {
+        if (value instanceof Array) {
+            if (value.length == 0) {
+                return true;
+            }
+            /*for (var i in value) {
+                let paramType = Registry.registeredParams["Point"];
+                if (!paramType.isValid(value[i])) {
+                    console.log("Does not contain a valid point");
+                    return false;
+                }
+            }*/
+            throw new Error("isValid is partially inoperable due to registeredParams deletion");
```

You won't need to throw this error

---

you'll need to delete the throw or else it will always throw the error